src/main/osd/osd_elements.c

```diff
@@ -1051,6 +1051,12 @@ static void osdElementMotorDiagnostics(osdElementParms_t *element)
     for (; i < getMotorCount(); i++) {
         if (motorsRunning) {
             element->buff[i] =  0x88 - scaleRange(motor[i], getMotorOutputLow(), getMotorOutputHigh(), 0, 8);
+#if defined(USE_ESC_SENSOR) || defined(USE_DSHOT_TELEMETRY)
+            if ( getEscRpm(i) < 700 ) {
```

Comment:
Where does this magic number come from? It should at least use a define to make it self explanatory.

Also, in Betaflight there are no spaces used on the inside of braces.

Reply:
Noted on the code style. Will update the PR.

With regards to magic value:
I found a curious behavior of the rpm telemetry. If motor is blocked, it twitches trying to start up. 
During this time the rpm will jump in from 0 to ~1500.
Eventually something stops the motor completely with a beep. is it ESC or Betaflight?

When motor gets disabled like that, the getEscRpm will return the last known value of rpm and this value stops being updated.

Experimentally, that value is never bigger than 700.

Also, the absolute minimum rpm i can get my motors to spin at is around 300-400, any lower and they stall.

Since the idle rpm is usually always bigger than 1000, it made sense to count anything lower than that as "stopped" in this case. 

I'm not too familiar with C. How should I define it?

Reply:
What I mean is to use a define - the equivalent of a `const` in C:

```
// Near the top of the file
#define MOTOR_STOPPED_THRESHOLD_RPM 700

[...]

if (getEscRpm(i) < MOTOR_STOPPED_THRESHOLD_RPM) {
```

Reply:
Ah I see. Thanks. Just pushed an update 👍 

Do you have any thoughts on the value itself?